lib/member_page.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+require 'scraped'
+
+class MemberPage < Scraped::HTML
+  field :birth_date do
+    date_from(
+      record_for('Datël')
+    ).to_s
+  end
+
+  field :birth_place do
+    record_for('Vendlindja:')
+  end
+
+  field :area do
+    record_for('Qarku:')
+  end
+
+  # TODO: This isn't so easy to extract as the text varies
+  # field :commissions do
+  #   puts line_for('Komisioni').split(/(Komisioni)/).drop(1).join('').tidy
+  # end
+
+  private
+
+  def record_for(str)
+    table.xpath(".//span[contains(., '#{str}')]")
+         .text
+         .split(':')
+         .drop(1)
+         .last
```

Comment:
This combination of `split(':').drop(1).last` is quite confusing. I _think_ what it's doing can be expressed slightly better as `split(':', 2)[1]`, but it's not clear whether you're expecting fields with more than one ':' or not. (If you are, you need to add a test case for them!)